src/Relativity.Server.Import.SDK.Samples/Relativity.Server.Import.SDK.Samples.csproj

```diff
@@ -12,8 +12,8 @@
     <PackageReference Include="Polly" Version="5.7.0" />
     <PackageReference Include="Relativity.Server.Import.SDK" Version="2.9.2" />
     <PackageReference Include="Relativity.Server.Kepler.Client.SDK" Version="2.15.6" />
-    <PackageReference Include="Relativity.Server.Logging.Interfaces.SDK" Version="3000.4.1" />
-    <PackageReference Include="Relativity.Server.OutsideIn.FI.Win32.SDK" Version="2023.4.0" />
+    <PackageReference Include="Relativity.Server.Logging.Interfaces.SDK" Version="3000.4.4" />

```

Comment:
If you're going to update the sample with the final Server 2024 SDKs, shouldn't you include all other updates? Also, explicit package references like Relativity.Server.Logging.Interfaces.SDK should be removed.

Reply:
@scott-parillo This is to give a quick fix to solve one of the [vulnerabilities ](https://jira.kcura.com/browse/REL-893257)that has passed the due date. We will upgrade all server SDKs once all SDKs are published to public artifacts.

Reply:
Removed Relativity.Server.Logging.Interfaces.SDK.